arraylias/alias.py

```diff
@@ -16,6 +16,42 @@
 from arraylias.exceptions import AliasError, LibraryError
 
 
+@functools.wraps(functools.lru_cache)
+def method_lru_cache(maxsize: Optional[int] = 128, typed: bool = False) -> Callable:
+
```

Comment:
Can you add a doc with some description here (including an explanation as to why this is necessary).

Reply:
The `wraps` makes the doc same as `functools.lru_cache` docs